src/cmd/flux-job.c

```diff
@@ -576,6 +577,13 @@ static struct optparse_subcommand subcommands[] = {
       0,
       timeleft_opts,
     },
+    { "last",
+      "SLICE",
+      "List my most recently submitted job ID(s)",
```

Comment:
Other subcommands in `flux job` reference jobids as `jobid(s)` or `job id(s)`. Instead of adding a third way, maybe we can settle on one (my preference is "jobid" "jobids". This is consistent with the `flux-job(1)` man page as well, though I haven't checked the rest of docs, --help, etc.)